RecoPixelVertexing/PixelTriplets/plugins/CAConstants.h

```diff
@@ -76,7 +76,7 @@ namespace caConstants {
   using OuterHitOfCellContainer = cms::cuda::VecArray<uint32_t, maxCellsPerHit>;
   using TuplesContainer = cms::cuda::OneToManyAssoc<hindex_type, maxTuples, 5 * maxTuples>;
   using HitToTuple = cms::cuda::OneToManyAssoc<tindex_type, -1, 4 * maxTuples>;  // 3.5 should be enough
-  using TupleMultiplicity = cms::cuda::OneToManyAssoc<tindex_type, 8, maxTuples>;
+  using TupleMultiplicity = cms::cuda::OneToManyAssoc<tindex_type, 10, maxTuples>;
```

Comment:
sorry for not following the definition... what is `10` (and what was `8`) ?

Reply:
the maximum number of "ones" in this case the number of possible multiplicities (counting from 0).
For simplicity I did not add an offset: so it is the maximal hit multiplicity -1